src/utils/msc_tree.cc

```diff
@@ -259,6 +259,7 @@ int InsertNetmask(TreeNode *node, TreeNode *parent, TreeNode *new_node,
 
             node->count++;
             node->netmasks = reinterpret_cast<unsigned char *>(malloc(node->count * sizeof(unsigned char)));
+            memset(node->netmasks, 0, (node->count *  sizeof(unsigned char)));
```

Comment:
@martinhsv This memset should probably go below the null check 2 lines down. Same for the other 2 in this PR.